es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super(30, 20);
```

Comment:
Instead of using hard coded values, we want this to be reusable. We should be utilizing our `width` property to pass in what the perimeter and area are.